examples/rich-text-sandbox/src/mdx.js

```diff
@@ -1,12 +1,12 @@
-// ../mdx/src/parse/index.ts
```

Comment:
I am not sure why when I ran build these files changed. @jeffsee55  Any ideas?

Reply:
Yeah I deleted this in another PR, my mistake. You can ignore this